hadoop-hdds/docs/content/security/SecuringTDE.md

```diff
@@ -66,3 +66,57 @@ via the encKey and while reading the clients will talk to Key Management
 Server and read the key and decrypt it. In other words, the data stored
 inside Ozone is always encrypted. The fact that data is encrypted at rest
 will be completely transparent to the clients and end users.
+
+### Using Transparent Data Encryption from S3G
+
+To use TDE from S3 interface, it can be done in 2 ways.
+
+####1. Create a bucket using shell under "/s3v" volume
+
+  ```bash
+  ozone sh bucket create -k encKey /s3v/encryptedBucket
+  ```
+####2. Create a link to an encrypted bucket under "/s3v" volume
+
+  ```bash
+  ozone sh bucket create -k encKey /vol/encryptedBucket
+  ozone sh bucket link  /vol/encryptedBucket /s3v/linkencryptedbucket
+  ```
+
+After this, all the keys created using s3g in the buckets will be encrypted.
+
+In non-secure mode, the user running the S3Gateway is the proxy user, 
+while in secure mode the user in Kerberos keytab is the proxy user. 
```

Comment:
```suggestion
while in secure mode the user accessing the S3Gateway is the proxy user. 
```

Reply:
Accessing means here keytab user implicitly?

Reply:
Yes, Bharat. I don't have a strong opinion here. Since keytab is used only for the first time and only access key and secret key will be used to access the S3G, I used "accessing the S3G".

Reply:
No, here it is explaining which user is proxying  so in secure case it is the kerberos keytab of s3g.

Reply:
Oh, yes, you are right. It will be the "ozone.s3g.kerberos.keytab.file" that will be the proxy user

Reply:
I have updated the suggestion